privaterelay/tests/mgmt_update_phone_remaining_stats_tests.py

```diff
@@ -35,40 +35,70 @@ def test_no_accounts_with_phones(db):
     update_phone_remaining_stats()
 
 
-def test_one_account_with_phones_checked_1_day_ago(phone_user):
+def test_one_account_with_phones_subscribed_3_day_ago_does_not_update_stats(phone_user):
+    profile = Profile.objects.get(user=phone_user)
+    datetime_now = datetime.now(timezone.utc)
+    profile.date_subscribed_phone = datetime_now - timedelta(3)
+    profile.save()
+
+    num_profiles_w_phones, num_profiles_updated = update_phone_remaining_stats()
+
+    profile.refresh_from_db()
+    assert profile.date_phone_subscription_checked == None
+    assert num_profiles_w_phones == 1
+    assert num_profiles_updated == 0
+
+
+def test_one_account_with_phones_checked_1_day_ago_does_not_update_stats(phone_user):
     profile = Profile.objects.get(user=phone_user)
     pre_update_datetime = datetime.now(timezone.utc) - timedelta(1)
     profile.date_phone_subscription_checked = pre_update_datetime
     profile.save()
 
-    updated_profiles = update_phone_remaining_stats()
+    num_profiles_w_phones, num_profiles_updated = update_phone_remaining_stats()
 
     profile.refresh_from_db()
     assert profile.date_phone_subscription_checked == pre_update_datetime
-    assert len(updated_profiles) == 0
+    assert num_profiles_w_phones == 1
+    assert num_profiles_updated == 0
 
 
-def test_one_account_with_phones_checked_31_day_ago_no_relay_number(phone_user):
+@patch(
+    "privaterelay.management.commands.update_phone_remaining_stats.get_phone_subscription_dates"
+)
+def test_one_account_with_phones_checked_31_day_ago_no_relay_number(
+    mocked_dates, phone_user
+):
     profile = Profile.objects.get(user=phone_user)
     pre_update_datetime = datetime.now(timezone.utc) - timedelta(31)
     profile.date_phone_subscription_checked = pre_update_datetime
     profile.save()
+    mocked_dates.return_value = (
```

Comment:
This tests a subscription that:

* Started 31 days ago
* Was checked when it started
* Ended just seconds ago

I think that means they have a lapsed subscription?

I'm curious about:

* A monthly user with an end date in the near future
* A monthly user with an end date in the far future
* A yearly user at the first month
* A yearly user after the first month but before the second
* A yearly user with a lapsed subscription (end date in the past)
* `free_phones` users at renewal